controls/bison_works.rb

```diff
@@ -0,0 +1,25 @@
+title 'Tests to confirm bison works as expected'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'bison')
+
+control 'core-plans-bison-works' do
+  impact 1.0
+  title 'Ensure bison works as expected'
+  desc '
+  '
+  plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+  end
+  
+  plan_pkg_ident = ((plan_installation_directory.stdout.strip).match /(?<=pkgs\/)(.*)/)[1]
+  plan_pkg_version = (plan_pkg_ident.match /^#{plan_origin}\/#{plan_name}\/(?<version>.*)\//)[:version]
```

Comment:
Could you add some comments explaining these? 
Hard to tell what they are doing at a glance.

Reply:
Are the comments in the control desciption ok?